examples/BUILD

```diff
@@ -110,6 +124,39 @@ genrule(
     cmd = "echo {} > $@"
 )
 
+jsonnet_to_json(
```

Comment:
This rule is redundant with the accompanying jsonnet_to_json_test rule.

Reply:
fixed now based on changes in latest master.